source/includes/_admin.md

```diff
@@ -185,7 +190,20 @@ no content
 
 **Request Body**
 
-An `AdminPayment' object
+An `AdminPayment` object containing, as a minimum, the new `transactionStatus` to be captured. The allowed values of the `transactionStatus` as well as what each `transactionStatus` means are explained below. 
+
+Please refer [Payment Transaction Resource](https://killbill.github.io/slate/?shell#payment-transaction-payment-transaction-resource) to understand the possible values of `transactionStatus`.
+
+
+## `AdminPayment` object
+
+| Name | Type | Generated by | Description |
+| ---- | -----| -------- | ------------ |
+| **lastSuccessPaymentState** | string | user | Last payment state |
+| **currentPaymentStateName** | string | user | New payment state |
+| **transactionStatus** | string | user | New transaction status  |
+
+Please refer [Payment States](https://github.com/killbill/killbill/blob/ff81c701e236310561e31624df18bab5788b0339/payment/src/main/java/org/killbill/billing/payment/core/sm/PaymentStateMachineHelper.java) information to understand the possible values of Payment State.
 
```

Comment:
One minor issue: In the slate docs, the "Resource" object is placed at the start of the section. For example, take a look at [Invoice Resource](https://killbill.github.io/slate/#invoice-invoice-resource), it is placed at the start of the [Invoice ](https://killbill.github.io/slate/#invoice)section.  Could you similarly move the "Admin Payment" resource to the start of the [Admin ](https://killbill.github.io/slate/#admin) section?

Reply:
Hi @reshmabidikar made these changes :)